pkg/providers/instancetype/metrics.go

```diff
@@ -64,6 +64,19 @@ var (
 			zoneLabel,
 		},
 	)
+	instanceTypeOfferingAvailableCapacityReservation = prometheus.NewGaugeVec(
```

Comment:
I wonder if we punt on this too. Ideally, we can just model this through our standard offering availabiliyt